### PR TITLE
Main

### PR DESCRIPTION
**Pull request by michaelc-yu**
Added artifact functionalities to AccuSleep_viewer.m

### CODE REVIEW

**Comment by Podothesheep** [comment]
Hey, sorry for the late response. I've been really busy with midterms for
the last couple of weeks. If I were to do this project, how long might the
whole process take? I am an undergrad and am generally really busy with
classes and a research lab, and although I'd like to contribute I also have
to be realistic with my schedule and time management.

**Comment by zekebarger** [comment]
I think the fix would take 1-2 hours at most. We'll need to add the boolean "predict" column to the config file, make sure AccuSleep_train_X only pays attention to classes where predict = True, and somehow enforce that the config file is ordered such that the predicted classes come before the non-predicted classes (otherwise the labels produced by the network could be in the wrong range). It's just a little tricky finding all the relevant bits of code that will need to change. For example, there's a function somewhere that checks whether a sufficient number of epochs of each state are in the training data, which would need to ignore non-predicted classes as well.

Let me know if you start working on this, otherwise there's a chance I'll get to it myself at some point :)

**Comment by Podothesheep** [comment]
Ok, I am up to do this then! I will keep you posted on this?

**Comment by Podothesheep** [comment]
It turns out this is kind of tricky for me. I think I will leave this
portion to you to do :)

**Comment by zekebarger** [comment]
Spring cleaning: closing this PR since the modification is better suited to AccuSleep X.
